mmv1/third_party/terraform/utils/validation.go

```diff
@@ -105,7 +105,7 @@ func validateRegexp(re string) schema.SchemaValidateFunc {
 		value := v.(string)
 		if !regexp.MustCompile(re).MatchString(value) {
 			errors = append(errors, fmt.Errorf(
-				"%q (%q) doesn't match regexp %q", k, value, re))
+				"%q (%q) doesn't match regexp %q for more info refer to https://cloud.google.com/compute/docs/naming-resources#resource-name-format", k, value, re))
```

Comment:
```suggestion
				"%q (%q) doesn't match regexp %q", k, value, re))
```

validateRegexp is used for other regexes as well, so we can't hardcode a specific link like this into the error message. Hypothetically we could allow callers to pass an optional link for more details but that is probably beyond the scope of this ticket.